stories/calendar/calendar.stories.js

```diff
@@ -4,75 +4,96 @@ export default {
     title: 'Components/Calendar',
     parameters: {
         description: `
-Commonly used as the contents of a popover when composing <code class="docs-code">date-picker</code>, rarely used on its own as a standalone component.
+The calendar component allows users to navigate to a single date, multiple days, whole week(s), or a date range, showing all time-related data within a given timeframe.
 
-> Looking for Date Picker? From a style perspective, a date-picker is simply a composition of our existing calendar, input group and popover, but contains no new styles of its own. Therefore, we consider it a pattern. [See the date picker pattern here](patterns/date-picker).
+It is usually paired with components: **Input Group** and **Popover** to comprise **Date Picker**, which is considered a design pattern as the date picker does not contain components of its own. The calendar component is rarely, if ever, used as a standalone component.
+        
+See **Date Picker** in the patterns section.
 
-### Structure
+## Usage
+**Use the calendar if:**
 
-Following classes describe block structure of calendar component.\n
-- <code class="docs-code">.fd-calendar</code> calendar main container, defines width of calendar
-  - <code class="docs-code">.fd-calendar\\_\\_header</code> header section of calendar
-    - <code class="docs-code">.fd-calendar\\_\\_navigation</code> container used for grouping row of navigation controls
-      - <code class="docs-code">.fd-calendar\\_\\_action</code> action buttons of calendar
-      - <code class="docs-code">.fd-calendar\\_\\_close-button--navigation</code> close button inside navigation for calendar on mobile devices without dialog header
-    - <code class="docs-code">.fd-calendar\\_\\_close-button</code> button inside dialog header for calendar on mobile devices
-  - <code class="docs-code">.fd-calendar\\_\\_content</code> calendar body container
-    - <code class="docs-code">.fd-calendar\\_\\_table</code> groups table elements representing calendar items
-      - <code class="docs-code">.fd-calendar\\_\\_group</code> groups calendar rows with the same destination
-        - <code class="docs-code">.fd-calendar\\_\\_row</code> groups calendar grid items in one row
-          - <code class="docs-code">.fd-calendar\\_\\_item</code> represents calendar grid item
-            - <code class="docs-code">.fd-calendar\\_\\_text</code> represents content of calendar grid item
-          - <code class="docs-code">.fd-calendar\\_\\_special-day</code> represents calendar grid item with special marker
+- You want the user to select a single date, multiple days, whole week(s), or a date range.
+- You want to display multiple months at once.
```

Comment:
or years as well

Reply:
Good catch! Added **years** to the sentence.